### ISSUE TITLE
Birthdate (age) validation is blocking enrollments

### ISSUE THREAD

**Issue by jlhogan** [ctoec/data-collection#1223]
## Context

currently we have a validation that requires birth dates to be <12 years ago, becuase that's the max age students can be in the programs funded by ECE funding streams. However (obviously) students will continue to age and so eventually _all_ enrollments will fail this validation. 
If we really wanted to validate this, we should be comparing birthdates to enrollment dates. But unblocking users is the primary goal here, so a separate ticket will address age/enrollment agegroup validations. 

## AC
- users can enroll a child who was 12 when they were in a program but are 13 now
- tldr; birthdate < 12 years validation is REMOVED
